nextflow.config

```diff
@@ -10,17 +10,18 @@ vim: syntax=groovy
  * name here.
  */
 
+//Placeholder for Igenomes
+params.igenomes_base = "./iGenomes"
+
 profiles {
+
   standard {
     includeConfig 'conf/base.config'
     includeConfig 'conf/uppmax.config'
     includeConfig 'conf/igenomes.config'
   }
   devel {
-    includeConfig 'conf/base.config'
-    includeConfig 'conf/uppmax.config'
```

Comment:
I think you need these other config includes still.. (also for aws)

Reply:
Thanks Phil! They have been added.